### PR TITLE
Integrating JUnit tests for WaveMaker application

### PR DESCRIPTION
**Pull request by abhi-va1289**
This document helps user's integrate JUnit tests in WaveMaker applications.

### CODE REVIEW

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+### PREREQUISITES 
+
+1. IDE (IntelliJ IDEA, Eclipse)
+2. WaveMaker Application
+3. Git
+4. Maven
+
+### CONFIGURATION
+
+1. Export the WaveMaker application as a zip file.
```
Comment (jvenugopal) [inline learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md:22]: link to the associated document in learn site.

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+2. Import this zip file into IDE (Use [WaveMaker workspace sync plugin](/learn/how-tos/synchronizing-wavemaker-apps-ides-beta)).
+3. Add the following dependencies in pom.xml file.
```
Comment (jvenugopal) [inline learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md:24]: Please mention under which section they need to be added like under <dependencies> etc.

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+```
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+  <import resource="file:src/main/webapp/WEB-INF/project-springapp.xml"/>
+</beans>
+```
+
+6. Go to folder src/test/java and create a folder structure com/<appName>/test/core
```
Comment (jvenugopal) [inline learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md:70]: not a folder structure, say package.

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+   @Test
+   public void getJsonData() {
+       Object jsonData = myJavaService.returnJSONData();
+       Assert.assertNotNull(jsonData);
+   }
+}
+
+```
+
+### To run unit tests before every commit
```
Comment (jvenugopal) [inline learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md:117]: remove this section.

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md (hunk 1 above)
Comment (abhi-va1289) [reply learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md]: Added the document to export WM project as a Zip

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md (hunk 2 above)
Comment (abhi-va1289) [reply learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md]: Added the section where the dependencies are to be added "under dependencies tag".

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md (hunk 3 above)
Comment (abhi-va1289) [reply learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md]: Changed from "Folder structure" to "Package"

File: learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md (hunk 4 above)
Comment (abhi-va1289) [reply learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md]: Removed the section

**Review by jvenugopal — APPROVED** [review]

**Comment by swethakundaram** [comment]
Since this is a duplicate, closing it.

**Pull request closed without merging**